docs/tutorial.rst

```diff
@@ -373,6 +375,10 @@ Note that if you use ``depends_on_past=True``, individual task instances
 will depend on the success of the preceding task instance, except for the
 start_date specified itself, for which this dependency is disregarded.
 
+The date range in this context is a ``start-date`` and optionally an ``end-date``,
```

Comment:
use `_` for `start_date` and `end_date` to match `airflow backfill -h` 's params